docs/modules/accesszone.rst

```diff
@@ -20,7 +20,7 @@ Requirements
 ------------
 The below requirements are needed on the host that executes this module.
 
-- A Dell PowerScale Storage system. Ansible 2.12, 2.13 or 2.14.
+- A Dell PowerScale Storage system. Ansible 2.13, 2.14 or 2.15.
```

Comment:
All rst files do not have updates requirements as doc_fragments/powerscale.py file

Reply:
Fixed.